vector/src/main/java/im/vector/app/features/MainActivity.kt

```diff
@@ -171,6 +181,13 @@ class MainActivity : VectorBaseActivity<ActivityMainBinding>(), UnlockedActivity
         } else if (intent.hasExtra(EXTRA_INIT_SESSION)) {
             setResult(RESULT_OK)
             finish()
+        } else if (intent.action == ACTION_ROOM_DETAILS_FROM_SHORTCUT) {
+            val roomId = intent.getStringExtra(EXTRA_ROOM_ID)
+            if (roomId?.isNotEmpty() == true) {
+                // TODO Add a trigger Shortcut to the analytics.
```

Comment:
I will handle that later

Reply:
worth a ticket so we don't forget?

Reply:
https://github.com/matrix-org/matrix-analytics-events/pull/69